### ISSUE TITLE
Page zoom shrinks the subject viewer

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#3336]
## Package
lib-classifier

## Describe the bug
Zooming into the Classify page, on any project, increases the text size (as expected) but shrinks the subject viewer. Zooming out shrinks the text but enlarges the subject.

https://user-images.githubusercontent.com/59547/174794439-4bef79f2-bd19-475e-8773-a9f1e22f510d.mov

## To Reproduce
1. Go to the Classify page on any project.
2. Use page zoom to zoom in.

## Expected behavior
The subject viewer should be enlarged, more like this screen recording.

https://user-images.githubusercontent.com/59547/174795413-19c200c2-d48f-4ed5-800e-f4c3b9548f97.mov

### Desktop (please complete the following information):
 - OS: MacOS 11.6.6
 - Browser: Firefox
 - Version: 101

## Additional context
The default layout sets the subject size to [`8fr`](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout#the_fr_unit), which will shrink to fit remaining space after the task panel has enlarged.
https://github.com/zooniverse/front-end-monorepo/blob/a1af82bf7da492fa3b004fec31e37cb221242dbf/packages/lib-classifier/src/components/Classifier/components/Layout/components/DefaultLayout/DefaultLayout.js#L23-L25
